backend/src/routes/image.ts

```diff
@@ -41,4 +44,26 @@ router.get("/image/:imageId", [verifyAuthToken], async (req: Request, res: Respo
   }
 });
 
+router.post(
+  "/image",
+  [verifyAuthToken],
+  upload.single("image"),
+  async (req: Request, res: Response) => {
+    try {
+      console.log("POST /image", req.file);
+      const imageId = await saveImage(req);
+      console.log("imageId", imageId);
+      return res.status(200).send({ imageId });
+    } catch (e) {
+      console.log(e);
+      if (e instanceof ServiceError) {
+        return res.status(e.status).send(e.displayMessage(true));
+      }
+      return res
+        .status(ServiceError.IMAGE_NOT_FOUND.status)
+        .send(ServiceError.IMAGE_NOT_FOUND.displayMessage(true));
```

Comment:
let's use the custom error handler by passing the error to next as done in POST mentee metnor routes

Example
https://github.com/TritonSE/ALUM-Mobile-Application/blob/efb91e96a30d57bba5169a31429438e609fb29c3/backend/src/routes/user.ts#L87